applications/nrf5340_audio/src/modules/audio_sync_timer.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2021, PACKETCRAFT, INC.
+ *
+ *  SPDX-License-Identifier: LicenseRef-PCFT
+ */
+
+#ifndef _AUDIO_SYNC_TIMER_H_
+#define _AUDIO_SYNC_TIMER_H_
+
+#include <zephyr/kernel.h>
+#include <stdint.h>
+
+/**
+ * @brief Capture a timestamp on the sync timer.
+ *
+ * @retval The current timestamp of the audio sync timer.
+ */
+uint32_t audio_sync_timer_capture(void);
+
+/**
+ * @brief Returns the last captured value of the sync timer.
```

Comment:
This Doxygen doc should include something about the capture event being I2S frame start